user/views/student.py

```diff
@@ -262,7 +261,7 @@ def put(self, request, *args, **kwargs) -> HttpResponse:
         try:
             params = json.loads(request.body.decode('utf8'))
         except json.JSONDecodeError as e:
-            return self.put_json_error(e)
+            return self.post_json_error(e)
 
```

Comment:
I noticed these convenience `put_json_error()` methods were missing so I replaced the calls to the `post_json_error()` methods which accomplish the same thing.